homeassistant/components/amcrest/camera.py

```diff
@@ -547,3 +586,22 @@ def _start_tour(self, start):
             log_update_error(
                 _LOGGER, "start" if start else "stop", self.name, "camera tour", error
             )
+
+    def _ptz_control(self, movement, travel_time):
+        """Move or zoom camera in specified direction."""
+        code = _ACTION[_MOV.index(movement)]
+        try:
+            kwargs = {"code": code, "arg1": 0, "arg2": 0, "arg3": 0}
+            if code in _MOVE_1_ACTIONS:
+                kwargs["arg2"] = 1
+            elif code in _MOVE_2_ACTIONS:
+                kwargs["arg1"] = kwargs["arg2"] = 1
```

Comment:
These statements should be outside (i.e., before) the `try` block. (I just got called out for this exact same thing on my PR. :smile: )

Reply:
fixed.   thanks,